framework/ui/view/musicalsymboliconcodes.h

```diff
@@ -0,0 +1,70 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2020 MuseScore BVBA and others
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program; if not, write to the Free Software
+//  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+//=============================================================================
+
+#ifndef MU_FRAMEWORK_MUSICALSYMBOLICONCODE_H
+#define MU_FRAMEWORK_MUSICALSYMBOLICONCODE_H
+
+#include <QObject>
+
+namespace mu {
+namespace framework {
+/**
+ * @brief The MusicalSymbolIconCode class simplifies access to the icons from the musical icon font
```

Comment:
Please, attach the link for this font

Reply:
MusicalSymbolIconCode depends on the current notation font(see qmltheme::musicalFont). The current font could be Leland, Bravura, etc. I think, there is no need to mention every font here.

Reply:
Yes, but it makes sense to mention our "default" font at least

Reply:
added